app/assets/javascripts/discourse/widgets/header-topic-info.js

```diff
@@ -57,6 +57,10 @@ createWidget("topic-header-participant", {
 export default createWidget("header-topic-info", {
   tagName: "div.extra-info-wrapper",
 
+  buildClasses(attrs) {
+    return this.showCategory(attrs.topic) ? "two-rows" : "";
```

Comment:
and `attrs.topic.category`  here ?

Reply:
Same as above. I can change it, but I do like how it reads.